### PR TITLE
Fix adding а batch to the queue by time trigger #13

### PR DESCRIPTION
**Pull request by ashulenko**
PR based on #13.
* Time counter is updated now when the batch is added to the queue. Before time counter was updated when adding the element (event) to the batch (ArrayList).
* This case is reproduced in versions >= 1.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ _by timeout_ and _by buffer size_.
 |1.3.*    |1.0.0                 |
 |1.9.0    |1.1.0                 |
 |1.9.0    |1.2.0                 |
+|1.9.0    |1.3.0                 |
```
Comment (mchernyakov) [inline README.md:23]: you could remove line 21-22 and leave just line #23

File: src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
     private final List<String> localValues;
     private final List<CompletableFuture<Boolean>> futures;
 
-    private volatile long lastAddTimeMillis = 0L;
+    private volatile long lastAddTimeMillis = System.currentTimeMillis();
```
Comment (mchernyakov) [inline src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java:26]: why do we need to set up it here?

Comment (ashulenko) [reply src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java:26]: Do you mean `System.currentTimeMillis()` instead `0L`?

File: README.md (hunk 1 above)
Comment (ashulenko) [reply README.md:21]: yeap, thanks
done

File: src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java (hunk 2 above)
Comment (mchernyakov) [reply src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java:26]: yes

Comment (ashulenko) [reply src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java:26]: I removed `if (lastAddTimeMillis == 0)` condition in `checkTime()` to optimize.
If take `0L` in current code version, than `checkTime()` gives true at first check `flushCondition()`, but it shouldn't.

Comment (mchernyakov) [reply src/main/java/ru/ivi/opensource/flinkclickhousesink/applied/ClickHouseSinkBuffer.java:26]: ok

**Review by mchernyakov — APPROVED** [review]

**Pull request merged**